discovery-provider/scripts/start.sh

```diff
@@ -106,20 +46,28 @@ if [[ "$audius_discprov_dev_mode" == "true" ]]; then
         echo "Finished running migrations"
     fi
 
-    # filter tail to server/worker/beat logs with
-    # docker exec -it <container> tail -f /var/log/discprov-server.log
-    # docker exec -it <container> tail -f /var/log/discprov-worker.log
-    # docker exec -it <container> tail -f /var/log/discprov-beat.log
-    ./scripts/dev-server.sh 2>&1 | tee >(logger -t server) &
```

Comment:
Remove the redirection of logs into `logger` and expose each process within it's own container logging to stdout, in line with the [12-factor app](https://12factor.net/logs).